### PR TITLE
Sky CMS case study

### PR DESCRIPTION
**Pull request by callerc1**
### Motivation

- https://trello.com/c/bI8RIuzU/32-sky-cms-case-study-port

### Test plan

- TODO: Write a test plan.

### Pre-merge checklist

- [ ] Documentation
- [ ] Unit tests
- [ ] Reviews
  - [ ] Code review
  - [ ] Design review
- [ ] Manual testing
  - [ ] Windows 7 IE 11
  - [ ] Windows 10 Edge
  - [ ] Mac OS El Capitan Safari
  - [ ] Mac OS El Capitan Chrome
  - [ ] Mac OS El Capitan Firefox
  - [ ] iOS 9 Safari
  - [ ] Android 6 Chrome
  - [ ] Listen to the site on a screen reader
  - [ ] Navigate the site using the keyboard only
- [ ] Tester approved

### CODE REVIEW

File: site/pages/our-work/case-study/sky/index.js [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+type CaseStudyFinancialTimesProps = {
+  contactUsURL: string,
+};
+
+const social = {
+  title: 'The proof is in the pudding',
+  description: 'Keeping up with the Financial Times',
+  metaImage: headerImage,
+  url: 'https://red-badger.com/our-work/case-study/financial-times',
```
Comment (martinhartt) [inline site/pages/our-work/case-study/sky/index.js]: Pretty sure these fields should be updated for Sky

File: site/pages/our-work/case-study/sky/index.js [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+};
+
+const FinancialTimesCaseStudy = ({ contactUsURL }: CaseStudyFinancialTimesProps) =>
```
Comment (martinhartt) [inline site/pages/our-work/case-study/sky/index.js]: The component name should also be updated :P

File: site/pages/our-work/case-study/sky/index.js [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+            },
+          ]}
+          itemClassName={styles.listBox__item}
+          labelClassName={styles.listBox__label}
+        />
+      </div>
+      <div className={styles.content__wrapper}>
+        <h2 className={styles.content__title}>
+          <span className={styles.content__redTitle}>
+            {"Let's make things better."}
```
Comment (martinhartt) [inline site/pages/our-work/case-study/sky/index.js]: Could this string be written without the curly brackets and quotes?

Comment (martinhartt) [inline site/pages/our-work/case-study/sky/index.js]: Also, maybe the single quote character should be ’ instead as its more commonly used

File: site/pages/our-work/case-study/sky/index.js [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+          to work. The team focused on improving the online help and support experience by enabling
+          customers to find content, resolve issues, navigate available services, and set up their
+          Sky equipment correctly.
+        </p>
+        <ContactBox />
+        <p className={styles.content__paragraph}>
+          Sky was already a leader in the use of agile methodologies and lean ways of working, and
+          were determined to work with a partner that would help them to advance these skills. To
+          complement their established structures, augmented by Sky’s UX/Design resources, we
+          brought in an n integrated team of UX, design, and technology to deliver rapidly. Bleeding
```
Comment (martinhartt) [inline site/pages/our-work/case-study/sky/index.js]: "we brought in an **n** integrated team"?

File: site/pages/our-work/case-study/sky/index.js [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+          they need it. Additionally, the help section was bolstered to handle spikes in activity;
+          saving customers from having to reach for the phone. So, when the football match starts
+          glitching, the Sky team can make sure that the solution is up on the site, or if one is
+          still being found, can put a pop up on the page to reassure customers are that they are
+          working to find the solution needed.
+        </p>
+        <p className={styles.content__paragraph}>
+          <img
+            src={cmsIllustrationImage}
+            alt="shopping cart"
```
Comment (martinhartt) [inline site/pages/our-work/case-study/sky/index.js]: This alt text doesn't represent the image

File: site/pages/our-work/case-study/sky/index.js [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+          made in less than three weeks, leaving room to implement further customer feedback.
+        </p>
+        <p className={styles.content__paragraph}>
+          The new diagnostic platform has had a huge effect. It tripled the number of issues
+          resolved through the online self-help journey, it doubled the customers serviced by the
+          advanced diagnostics, and it’s serviced half a million customers to date.
+        </p>
+      </div>
+    </div>
+    <WhatToReadNext currentPage="financialTimes" />
```
Comment (martinhartt) [inline site/pages/our-work/case-study/sky/index.js]: This component should be updated to include a sky thumbnail perhaps

File: site/pages/our-work/case-study/sky/index.js [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+import authorQuoteImage from './images/quote-author.jpg';
+
+import ListBox from '../../../../components/list-box';
+import Quote from '../../../../components/quote';
+
+import WhatToReadNext from '../what-to-read-next';
+import ContactUs from '../../../../slices/contact-us-slice';
+import ContactBox from '../../../../components/contact-box';
+
+type CaseStudyFinancialTimesProps = {
```
Comment (martinhartt) [inline site/pages/our-work/case-study/sky/index.js]: This type should be renamed

Comment (callerc1) [inline site/pages/our-work/case-study/sky/index.js]: cmd + C 👍

**Comment by chris-brookes** [comment]
@callerc1 - 
**1. When h2 headers are between text paragraphs, the padding above the header should be the same, or greater, than the padding below. In mobile view, can you please increase the margin above the header "Do the right thing. Do the thing right." (make sure you do not increase the margin between the other paragraphs)**

screen shot 2017-08-09 at 14 34 40


**2. All views - Increase the font size of the Results table title to 24px.**


screen shot 2017-08-09 at 14 34 53

**Comment by callerc1** [comment]
Style amends have been made @chris-brookes along with peer review updates @martinhartt

**Comment by chris-brookes** [comment]
@callerc1 
**The contact box is getting too squashed in tablet view. The width % should not be set in tablet view.**

screen shot 2017-08-09 at 16 48 11


 
**The box should go further into the text, as like the design.**


 
screen shot 2017-08-09 at 16 51 22

**Comment by chris-brookes** [comment]
@callerc1 - That contact box set width still needs to be set for desktop view, it only needed to be removed from tablet view.

**Comment by martinhartt** [comment]
All good except for the alt text for the first image in the SkyCmsCaseStudy component

**Comment by callerc1** [comment]
@chris-brookes @martinhartt all good now?

**Comment by chris-brookes** [comment]
@callerc1 - Yeah the fix for me looks good, buuuuuut I've just found another one. (Sorry it is annoying when you keep finding bugs after you think you've finished)

FireFox helped me get this one, you don't notice it if you're not directly looking for it in the other browsers. The header image and the background colour do not match. 

The background colour in Zeplin is different to the one in the code, but even if you change to the colour from Zeplin, it still doesn't match the image colour. I think we'll have to speak to Jun to get the correct image colour.

**Comment by chris-brookes** [comment]
@callerc1 - Change the top and bottom margins on all H2 headers used in the main text content (3 of them). Mobile view only. No change to tablet and desktop margins.

Top margin 50px
Bottom margin 20px

eg.
screen shot 2017-08-11 at 11 38 44

**Pull request merged**